src/components/main-nav/main-nav.js

```diff
@@ -6,5 +6,50 @@ export default class MainNav {
   constructor (node) {
     this.node = document.querySelector(node)
     this.node.innerHTML = template(mainNavData)
+    this.items = document.querySelectorAll('.menu__item')
+    this.links = document.querySelectorAll('.menu__link')
+    this.deploySubmenuEvent()
+    this.deploySubmenuEventLarge()
+    this.toggleMenuEvent()
+  }
+
+  deploySubmenuEvent () {
+    this.items.forEach((_, index, array) => {
+      array[index].addEventListener('click', e => {
```

Comment:
Instead of passing `_` as the first parameter, you can actually use a variable name as `item` in order to access the current iteration item: `item.addEventListener('click'...` instead of `array[index].addEventListener('click'...`